config/config_network.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
```

Comment:
no longer required, WiFi Manager is used to connect to network or create an Accesspoint